localize/commands.py

```diff
@@ -135,6 +136,12 @@ def pull(conf):
   # If there are any errors display them to the user
   if errors:
     for error in errors:
-      print(Fore.RED+error+Style.RESET_ALL)
+      print(Fore.RED + error + Style.RESET_ALL)
   else:
     sys.exit(Fore.GREEN + 'Successfully pulled ' + str(len(conf['pull']['targets'])) + ' file(s) to Localize!' + Style.RESET_ALL)
+
+def check_and_return_lang_format(filename, type):
+  if filename.count('.') != 2:
```

Comment:
Filename should be `<name>.<lang>.<format>`, for example `project_name.ru.json`
If filename will be `project_name.ru_json` (< 1 dot) or `project_name.example.ru.json` (> 2 dots) 'if statement' will return `sys.exit` with instructions. And if filename will be `project_name.abc.json` (wrong lang) or `project_name.eng.abc` (wrong format) user will get exception from server.

BTW fileformat i got from here: https://github.com/Localize/localize-cli/blob/master/localize/commands.py#L31,L34